stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/internal/ExtraDataValidator.kt

```diff
@@ -166,7 +166,7 @@ internal class ExtraDataValidator(
 
     private companion object {
 
-        private val reservedInChannel = arrayOf(
```

Comment:
I would move it inside the class instead of having functions because we gonna produce that array on each API call which is getting validated

Reply:
Indeed, the companion object is unnecessary in this class